.watchmanconfig

```diff
@@ -0,0 +1,3 @@
+{
+  "ignore_dirs": ["tmp", "dist"]
```

Comment:
What is the purpose of this file?

Reply:
This file comes from ember-cli. I keep it for maintenance reasons.